### PR TITLE
Staff course list component

### PR DESCRIPTION
**Pull request by pconrad**
In this PR, we add a basic course list component, as a starting point for the F20 student teams that are working on the basic UI for course search results.

This is only a starting point; the eventual UI might or might not be based on `BootstrapTable`.  This is really just to illustrate how you might go about building a better UX than the one that currently just displays the raw JSON.

### CODE REVIEW

File: javascript/src/main/pages/Home/Home.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React from "react";
 import { useState } from "react";
 import { Jumbotron } from "react-bootstrap";
-import { useAuth0 } from "@auth0/auth0-react";
-import { Redirect } from "react-router-dom";
 import BasicCourseSearchForm from "../../components/BasicCourseSearch/BasicCourseSearchForm";
+import BasicCourseTable from "../../components/BasicCourseSearch/BasicCourseTable";
 import JSONPrettyCard from "../../components/Utilities/JSONPrettyCard";
```
Comment (scottpchow23) [inline javascript/src/main/pages/Home/Home.js:6]: We should try to stick to absolute paths instead of relative ones.

**Review by scottpchow23 — APPROVED** [review]
One nitpick on include paths but otherwise it looks good to me!

File: javascript/src/main/pages/Home/Home.js (hunk 1 above)
Comment (pconrad) [reply javascript/src/main/pages/Home/Home.js]: Fixed

**Review by scottpchow23 — APPROVED** [review]

**Review by tkomarlu — APPROVED** [review]
LGTM

**Pull request merged**